feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/transaction/client/main/module.xml

```diff
@@ -23,6 +23,10 @@
   -->
 
 <module xmlns="urn:jboss:module:1.5" name="org.wildfly.transaction.client">
+    <properties>
+        <property name="jboss.api" value="unsupported"/>
+    </properties>
```

Comment:
This either needs to be "private" or the PR and WFLY JIRA needs to be withdrawn and this becomes a JBEAP-only thing. There is no such thing as "unsupported" in community. It's an EAP-only value that means public in community but not supported in the product. With "supported" very much meaning in the contractual sense.

If it's not meant for direct end user use in community then the value should be "private".